bin/fmt

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -u
+
+while IFS= read -r line; do dirs+=("$line"); done <<< "$(go list -f \{\{.Dir\}\} ./...)"
+
+# go list will list all subdirectories and goimports acts recursively.  This
+# results in certain files being reported multiple time.  Therefore, we must
+# dedup them. We also ignore protobuf auto-generated code.
+files=$(bin/goimports -l "${dirs[@]}" | sort -u | grep -v pb.go)
+
+if [ -n "$files" ]; then
+  for file in "${files[@]}"; do
+    bin/goimports -d "$@" "$file"
```

Comment:
I assume `bin/fmt` will get called by #1 . You still need then to add `bin/goimports` in this repo.